test/ctmrg/partition_function.jl

```diff
@@ -44,8 +43,9 @@ Implements the 2D classical Ising model with partition function
 \\mathcal{Z}(\\beta) = \\sum_{\\{s\\}} \\exp(-\\beta H(s)) \\text{ with } H(s) = -J \\sum_{\\langle i, j \\rangle} s_i s_j
 ```
 """
-function classical_ising(; beta=log(1 + sqrt(2)) / 2, J=1.0)
+function classical_ising(beta=log(1 + sqrt(2)) / 2, J=1.0)
```

Comment:
I think not using the kwargs is a bit breaking with the convention we have for all other models. Does it matter?

Reply:
I was planning to try out computing some stat mech observables using AD by differentiating with respect to temperature, at which point I made this a normal argument. But then I didn't actually do that, so it doesn't matter here. If you prefer I can switch it back.

Reply:
Maybe? It clearly also doesn't matter too much, I do like some consistency